docs/content/1.guide/7.plugins.md

```diff
@@ -27,6 +27,34 @@ export default defineNitroPlugin((nitroApp) => {
 })
 ```
 
+**Example:** Response Modification
+
+You can use plugins to register a hook that modifies the renderer response.
+
+```ts
+export default defineNitroPlugin((nitro) => {
+
+  nitro.hooks.hook('render:response', (response) => {
+    // Modify the renderer response here
+    console.log(response)
+  })
+})
+```
+
+**Example:** Graceful Shutdown
```

Comment:
This is more generic example to be first imo

Reply:
I accidentally push to the wrong branch https://github.com/unjs/nitro/compare/main...Hebilicious:nitro:docs/nitro-plugins